common/djangoapps/third_party_auth/management/commands/saml.py

```diff
@@ -25,9 +25,16 @@ def handle(self, *args, **options):
         log = logging.getLogger('third_party_auth.tasks')
         log.propagate = False
         log.addHandler(log_handler)
-        num_changed, num_failed, num_total = fetch_saml_metadata()
+        num_changed, num_failed, num_total, failure_messages = fetch_saml_metadata()
         self.stdout.write(
             "\nDone. Fetched {num_total} total. {num_changed} were updated and {num_failed} failed.\n".format(
                 num_changed=num_changed, num_failed=num_failed, num_total=num_total
             )
         )
+
+        if num_failed > 0:
+            raise CommandError(
+                "Command failed while fetching saml metadata.\n\n{failures}".format(
```

Comment:
The command isn't really failing here, so how about we say something like `Command finished with the following exceptions: ...`